src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useEffect, useState } from "react"
+import { BrowserRouter as Router } from "react-router-dom"
+import Layout from "antd/lib/layout"
+import Menu from "antd/lib/menu"
+import { Routes } from "./Routes"
+import "antd/dist/antd.css"
+import "./App.scss"
+import { QueryClient, QueryClientProvider } from "react-query"
+
+const { Header, Content, Footer } = Layout
+const queryClient = new QueryClient()
+
+export const App: React.FC = (): ReactElement => {
+  const [date, setDate] = useState(0)
+  const getYear = () => setDate(new Date().getFullYear())
+
+  useEffect(() => {
+    getYear()
```

Comment:
This should either be called *updateYear* or you do a *setYear* inside the *useEffect*. Otherwise it's misleading.